assets/custom.css

```diff
@@ -339,6 +339,15 @@ textarea#description {
 .ui.ginsearch > .results > :last-child {
   border-radius: 0em 0em 0.28571429rem 0.28571429rem;
 }
+#history {
+  margin-bottom: 20px;
+}
+.time-tiny {
```

Comment:
If we add a linebreak between badge and date/time, we can do without small fonts. Please remove the `.time/.date-tiny`.